semgrep-core/Parsing/other/yaml_to_generic.mli

```diff
@@ -1,2 +1,7 @@
-val program: string -> AST_generic.program
+type env = {
```

Comment:
this should not be exposed in the mli. This is an internal data structure that you use to pass
around information in recursive function, but the interface of the module should remain simple.